src/Test/PdbUtilities/Shared/SymUnmanagedReaderExtensions.cs

```diff
@@ -322,7 +63,7 @@ internal static ImmutableArray<string> GetLocalVariableSlots(this ISymUnmanagedM
             Debug.Assert((offset < 0) || scope.IsInScope(offset, isScopeEndInclusive));
 
             // apply action on locals of the current scope:
-            var locals = GetLocalsInternal(scope);
+            var locals = scope.GetLocals();
             if (locals != null)
```

Comment:
This can be further simplified. GetLocals doesn't return null anymore, but an empty array.